packages/block-library/src/group/edit.js

```diff
@@ -63,11 +66,31 @@ function GroupEditControls( { tagName, onSelectTagName } ) {
 					{ label: '<article>', value: 'article' },
 					{ label: '<aside>', value: 'aside' },
 					{ label: '<footer>', value: 'footer' },
+					{ label: '<nav>', value: 'nav' },
 				] }
 				value={ tagName }
-				onChange={ onSelectTagName }
+				onChange={ ( value ) => {
+					setAttributes( {
+						tagName: value,
+						ariaLabel: value === 'nav' ? ariaLabel : undefined,
```

Comment:
This is going to override already existing aria labels when the tag name is changed to and from `nav`

Reply:
This should work like this:

- When changed TO a `nav` element: existing aria-Label is maintained
- When changed FROM a `nav` element: aria-Label is removed

Reply:
Is this expected though? I am torn, it would be good to have some user testing.

If I have a group block and I use the transform, either to another block like the details or to a row or stack, the label is kept.

Reply:
If the UI was always available, I don't think it would be expected.
Without the UI, it is more difficult to discover both that the label is removed, or that it is kept.